main.go

```diff
@@ -87,20 +59,39 @@ func parseAuthFile(filename string) ([]*sshmux.User, error) {
 }
 
 func main() {
-	// Config
-	if len(os.Args) != 2 {
-		usage()
-		return
+	viper.SetDefault("address", ":22")
+	viper.SetDefault("hostkey", "hostkey")
+	viper.SetDefault("authkeys", "authkeys")
+
+	viper.SetConfigName("sshmuxd")
+	viper.AddConfigPath(".")
+	viper.AddConfigPath("$HOME/.sshmuxd")
+	viper.AddConfigPath("/etc/sshmuxd/")
```

Comment:
Could we maintain the option to support user-supplied configuration filename? Something like:

```go
import "flag"
userConfFile := flag.String("configFile", "", "user-supplied configuration file to use")

if userConfFile == "" {
    // Try $HOME and /etc
} else {
    // Use supplied config
}
```

It's kind of convenient for debugging purposes to be able to have more than one configuration file in a directory.